grafana_notification_policies/client_grafana_notification_policy.go

```diff
@@ -0,0 +1,80 @@
+package grafana_notification_policies
+
+import (
+	"fmt"
+	"github.com/logzio/logzio_terraform_client/client"
+)
+
+const (
+	grafanaNotificationPolicyServiceEndpoint = "%s/v1/grafana/api/v1/provisioning/policies"
+
+	grafanaNotificationPolicyResourceName   = "grafana notification policy"
+	operationSetGrafanaNotificationPolicy   = "SetNotificationPolicyTree"
+	operationGetGrafanaNotificationPolicy   = "GetNotificationPolicyTree"
+	operationResetGrafanaNotificationPolicy = "ResetNotificationPolicyTree"
+
+	MatchTypeEqual     MatchType = "="
+	MatchTypeNotEqual  MatchType = "!="
+	MatchTypeRegexp    MatchType = "=~"
+	MatchTypeNotRegexp MatchType = "!~"
+)
+
+type GrafanaNotificationPolicyClient struct {
+	*client.Client
+}
+
+type GrafanaNotificationPolicyTree struct {
+	GroupBy        []string                    `json:"group_by,omitempty"`
+	GroupInterval  string                      `json:"group_interval,omitempty"`
+	GroupWait      string                      `json:"group_wait,omitempty"`
+	Provenance     string                      `json:"provenance,omitempty"`
+	Receiver       string                      `json:"receiver,omitempty"`
+	RepeatInterval string                      `json:"repeat_interval,omitempty"`
+	Routes         []GrafanaNotificationPolicy `json:"routes,omitempty"`
+}
+
+type GrafanaNotificationPolicy struct {
+	Continue          bool                        `json:"continue"`
+	GroupBy           []string                    `json:"group_by,omitempty"`
+	GroupInterval     string                      `json:"group_interval,omitempty"`
+	GroupWait         string                      `json:"group_wait,omitempty"`
+	MuteTimeIntervals []string                    `json:"mute_time_intervals,omitempty"`
+	ObjectMatchers    MatchersObj                 `json:"object_matchers,omitempty"`
```

Comment:
Is there a way to include multiple matchers? Take for example a case where you want to route the notifications based on values of 2 or more labels

Reply:
Yes, the matcher object holds a list of strings